images/linux/scripts/tests/Tools.Tests.ps1

```diff
@@ -0,0 +1,19 @@
+Import-Module "$PSScriptRoot/../helpers/Tests.Helpers.psm1" -DisableNameChecking
```

Comment:
Do we really need to import it in every test file?
We have already import it in `RunAll-Tests.ps1` and in `invoke_tests()`. It should be enough

Reply:
Yes, I agree, removed extra importing. Thanks!